_static/js/custom.js

```diff
@@ -250,6 +250,16 @@ $(document).ready(() => {
     registerOnScrollEvent(mediaQuery);
   });
 
+  // Add line-break suggestions to code links
+  const referenceInternalList = document.body.querySelectorAll(".reference.internal");
```

Comment:
We use single quotes in this file.

I think we should target specifically the last `<ul>` of `.wy-menu-vertical`, so it only affects the class reference pages. Also using `body` doesn't seem necessary here.